modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -391,46 +394,40 @@ protected Assignment getAssignment(User.ID userID, Application.Name applicationN
                     //todo: change so this doesn't follow the 'read then write' Cassandra anti-pattern
                     assignment = generateAssignment(experiment, userID, context, selectBucket, bucketList, currentDate);
                     assert assignment.getStatus() == Assignment.Status.NEW_ASSIGNMENT :
-                            new StringBuilder("Assignment status should have been NEW_ASSIGNMENT for ")
-                                    .append("userID = \"").append(userID).append("\", experiment = \"")
-                                    .append(experiment).append("\"").toString();
+                            "Assignment status should have been NEW_ASSIGNMENT for " +
```

Comment:
I think string builder should be used here rather than changing it to string concatenation.